script/BaseData.s.sol

```diff
@@ -55,130 +55,126 @@ contract BaseData is Script {
         morphTestnet: 2810
     });
 
-    address private TEMP_GNOSIS_SAFE;
-    address private TEMP_PROXY_CONTROLLER;
+    address private TEMP_YN_SECURITY_COUNCIL;
 
     function setUp() public virtual {
-        TEMP_GNOSIS_SAFE = makeAddr("gnosis-safe");
-        TEMP_PROXY_CONTROLLER = makeAddr("proxy-controller");
+        TEMP_YN_SECURITY_COUNCIL = makeAddr("temp-yn-security-council");
 
         // NOTE: All the LZ Endpoints and EIDs are picked up from their docs
         // at https://docs.layerzero.network/v2/developers/evm/technical-reference/deployed-contracts
 
         // mainnets
         __chainIdToData[__chainIds.mainnet] = Data({
-            OFT_OWNER: TEMP_GNOSIS_SAFE,
-            TOKEN_ADMIN: TEMP_GNOSIS_SAFE,
-            PROXY_ADMIN: TEMP_PROXY_CONTROLLER,
+            OFT_OWNER: TEMP_YN_SECURITY_COUNCIL,
```

Comment:
Need yn security council addresses for mainnet, manta and fantom.

Reply:
- added mainnet address in [587aaa9](https://github.com/yieldnest/yieldnest-cross-chain/pull/11/commits/587aaa9f6c25973087c66e4da7b7ff737e960bd2)
- not supporting manta and fantom for now.